hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -26,6 +30,47 @@ export const cartReducerName = "cart";
 export const initialState = cartAdapter.getInitialState(extraState);
 export type CartState = typeof initialState;
 
+// Thunk
+interface RejectValue {
+    status: number;
+    message: any;
```

Comment:
Eventually we'll want to figure out what this is. I know this endpoint doesn't return a string, so this should probably be renamed `data` or something and given an appropriate type definition.

The actual errors are the Validation Errors from [here](https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/hardware/serializers.py#L167). I think the response will be something like

```typescript
interface OrderCreateError {
  non_field_errors: string[];
}
```